gramps/gui/editors/displaytabs/embeddedlist.py

```diff
@@ -42,6 +42,7 @@
 # Gramps classes
 #
 #-------------------------------------------------------------------------
+from gramps.gui.widgets.cellrenderertextedit import CellRendererTextEdit
```

Comment:
Use a relative import here.

Reply:
FWIW I think this policy needs adjustment.  Relative imports seem to me to be fine when you are importing from the same directory (one '.').  But I think they are confusing and hard to read if you are going deeper.  If we ever have to restructure our sources again, the multi-dot syntax will NOT make for an easy search/replace.

But I fixed it.